src/include.am

```diff
@@ -8,7 +8,8 @@ src_libwolftpm_la_SOURCES      = \
 								src/tpm2.c \
 								src/tpm2_packet.c \
 								src/tpm2_tis.c \
-								src/tpm2_wrap.c 
+								src/tpm2_wrap.c \
+                                src/tpm2_linux.c
```

Comment:
Check indent. Feel free to adjust ones above to use spaces (we prefer spaces).

Reply:
The ones above were not spaces, fixed ;-)